ci/apply_folder.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
```

Comment:
Those bash scripts should be in a dedicated directory - for example, prow_commands or something like that